src/providers/azure.rs

```diff
@@ -515,6 +289,232 @@ impl Drop for RegionLauncher {
     }
 }
 
+/// Available regions to launch VMs in.
+///
+/// See https://azure.microsoft.com/en-us/global-infrastructure/locations/ for more information.
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+pub enum Region {
```

Comment:
Maybe it'd be an idea to put these in a submodule?

Reply:
I thought about another submodule, but decided against - it's probably not worth the extra source file + indirection.